.ado/templates/e2e-test-job.yml

```diff
@@ -66,7 +58,7 @@ jobs:
       - task: CmdLine@2
         displayName: run-windows
         inputs:
-          script: react-native run-windows --no-packager --arch ${{ parameters.BuildPlatform }} --release --logging --msbuildprops BaseIntDir=$(BaseIntDir)
+          script: react-native run-windows --no-packager --arch ${{ parameters.BuildPlatform }} --release --logging --msbuildprops BaseIntDir=$(BaseIntDir),UseDotNetNativeToolchain=false
```

Comment:
>UseDotNetNativeToolchain=false [](start = 156, length = 30)

is this necessary? Note that some platforms like wcos won't run with this setting (there is no managed framework installed)
If we have to keep this as is, please add a comment why, and file an issue to remove it

Reply:
I thought this would resolve https://github.com/microsoft/react-native-windows/issues/4122, but it doesn't. I'll remove it.